arthur_bench/run/utils.py

```diff
@@ -40,8 +40,11 @@ def _load_suite_from_args(
     if requires_reference:
         if reference_column is None and reference_output_list is None:
             raise UserValueError(
-                "scoring method requires reference data but no reference data was "
-                "provided"
+                "scoring method requires Reference Outputs but no"
+                "Reference Output was provided. "
+                "Please provide a reference_output_list or a reference_column"
```

Comment:
this is probably gonna show up in the error message as 
"Please provide a reference_output_list or a reference_columnname (if a DataFrame was used)."

Reply:
gotta manually add some spaces to some of these newline strings